tools/templates/L10N/IT

```diff
@@ -0,0 +1,411 @@
+# This file contains the Italian translation of features of the
+# Raku Programming Language.
+#
+# See tools/build/L10N/CORE for more explanation.
+
+# KEY           TRANSLATION
+block-default   predefinito
```

Comment:
Italian, same as Spanish, has gendered adjectives. Going for masculine, here. But it's a judgement call.

Reply:
It's referring to:
````raku
default {
    say "default action in 'when' sequence";
}
````

Reply:
OK, then it's action, "azione", that's female. Changing it.